app/api/customer/route.js

```diff
@@ -0,0 +1,28 @@
+import { NextResponse } from "next/server";
```

Comment:
Don't we have this route already?

Reply:
That one is to get all customers, i only need a specific one